src/microbe_stage/systems/ProcessSystem.cs

```diff
@@ -260,6 +289,7 @@ public sealed class ProcessSystem : AEntitySetSystem<float>
         }
 
         result.FinalBalance = result.TotalProduction - result.TotalConsumption;
+        result.ConservativeFinalBalance = result.TotalProduction - result.TotalConsumption;
```

Comment:
Is there a typo on this line? This uses the same variables to calculate as the non-conservative version.

Reply:
Yep that is a typo. Good catch.